gbpservice/nfp/configurator/agents/generic_config.py

```diff
@@ -131,10 +171,14 @@ def _get_driver(self, service_type):
         return self.drivers[service_type]()
 
     def _notification(self, data):
+        LOG.info(" Sending notification=%s " % (data))
         event = self.sc.new_event(
             id='NOTIFICATION_EVENT', key='NOTIFICATION_EVENT', data=data)
         self.sc.poll_event(event)
 
+    def handle_poll_event(self, ev):
+        pass
```

Comment:
First decorator methods for poll event will be called and at the last this function gets called. Will remove this function